src/mame/ussr/jukumouse.h

```diff
@@ -0,0 +1,30 @@
+// license: BSD-3-Clause
+// copyright-holders:Märt Põder
+
+#ifndef MAME_USSR_JUKU_H
```

Comment:
`MAME_USSR_JUKUMOUSE_H` (here and end of file)

Reply:
Renamed, thank you, @plaes!